framework/src/st_things/things_stack/cloud/cloud_manager.c

```diff
@@ -1485,9 +1484,13 @@ static OCStackResult register_server_into_cloud(es_cloud_prov_data_s *event_data
 		signed_up_data = NULL;
 	}
 
+	if (g_cloud_ip[0] == NULL || g_cloud_port[0] == NULL) {
+		THINGS_LOG_E(TAG, "g_cloud_ip is NULL or g_cloud_port is NULL");
+		return res;
+	}
 	int port = atoi(g_cloud_port);
-	if (g_cloud_ip[0] == NULL || g_cloud_port[0] == NULL || 0 >= port || port > 65535) {
-		THINGS_LOG_E(TAG, "Cloud info is invalid.(g_cloud_ip=%s, g_cloud_port=%s, port=%d)", g_cloud_ip, g_cloud_port, port);
+	if (0 >= port || port > 0xffff) {
```

Comment:
log_in_out_to_cloud also using same logic here, why don't you fix it together?

Reply:
It was not intended to check everything.
I had to change only the visible dirty logic.
But that part has also been revised.